modules/javafx.graphics/src/main/java/javafx/scene/PointLight.java

```diff
@@ -26,18 +26,35 @@
 package javafx.scene;
 
 import com.sun.javafx.scene.DirtyBits;
-import com.sun.javafx.scene.NodeHelper;
 import com.sun.javafx.scene.PointLightHelper;
 import com.sun.javafx.sg.prism.NGNode;
 import com.sun.javafx.sg.prism.NGPointLight;
+
+import javafx.beans.property.DoubleProperty;
 import javafx.scene.paint.Color;
+import javafx.scene.paint.PhongMaterial;
 
 /**
- * Defines a point light source object. A light source that has a
- * fixed point in space and radiates light equally in all directions
- * away from itself.
+ * A light source that radiates light equally in all directions away from itself. The location of the light
+ * source is a single point in space. Any pixel within the range of the light will be illuminated by it,
+ * unless it belongs to a {@code Shape3D} outside of its {@code scope}.
+ * <p>
+ * The light's intensity can be set to decrease over distance by attenuating it. The attenuation formula
+ * <p>
+ * {@code attn = 1 / (ca + la * dist + qa * dist^2)}
+ * <p>
+ * defines 3 coefficients: {@code ca}, {@code la}, and {@code qa}, which control the constant, linear, and
+ * quadratic behaviors of intensity falloff over distance, respectively. The effective color of the light
+ * at a given point in space is {@code color * attn}. It is possible, albeit unrealistic, to specify negative
+ * values to attenuation coefficients. This allows the resulting attenuation factor to be negative, which
+ * results in the light's color being subtracted from the material instead of added to it, thus creating a
+ * "shadow caster".
```

Comment:
Can you think of any problems that might arise by supporting negative coefficients? If not, then this seems fine.

Reply:
No, I've tested the functionality and it seemed to be working well.